yast/lib/suse/connect/status.rb

```diff
@@ -0,0 +1,28 @@
+require 'suse/toolkit/shim_utils'
+
+module SUSE
+  module Connect
+    # The System Status object provides information about the state of currently installed products
+    # and subscriptions as known by registration server.
+    # At first it collects all installed products from the system, then it gets its `activations`
+    # from the registration server. This information is merged and printed out.
+    # rubocop:disable ClassLength
+    class Status
+      include SUSE::Toolkit::ShimUtils
+
+      def initialize(client_params)
+        @client_params = client_params
+      end
+
+      def activated_products
```

Comment:
does this need `_set_verify_callback` ?

Reply:
`Status` is instantiated via `SUSE::Connect::YaST.status()` there's `_set_verify_callback` call there.